sirepo/agent_supervisor_api.py

```diff
@@ -71,11 +50,7 @@ def request(method, uri, token, data=None, json=None):
         json=json,
         data=data,
         verify=sirepo.job.cfg().verify_tls,
-        headers=PKDict(
-            {
-                sirepo.util.AUTH_HEADER: _AUTH_HEADER_PREFIX + token,
-            }
-        ),
+        headers=sirepo.http_util.auth_header(token),
```

Comment:
This should be encapsulated in sirepo.tornado.AuthHeaderRequestHandler, since it is the one interpreting the requests. It's find to also have it in sirepo.http_util, but from the perspective of this module, it should be in the superclass.

Reply:
Should the same be done for raydata?

https://github.com/radiasoft/sirepo/pull/7146/files#diff-a23ec8b07403deef35f70ca014733d2a3b9e582c40cdc3b7e862a04201e72616R87 

Should it be a class method that creates it?

Reply:
I think we should put in the class for encapsulation purposes. It could also be in sirepo.http_util for the actual work for things that aren't subclasses.

Reply:
I think we should put in the class for encapsulation purposes. It could also be in sirepo.http_util for the actual work for things that aren't subclasses.

Reply:
322a67240